kafka/kafka_consumer_test.go

```diff
@@ -366,7 +366,7 @@ var _ = Describe("Kafka Extension", func() {
 				defer client.StopConsuming()
 				go client.ConsumeLoop()
 
-				Eventually(func() []*logrus.Entry {
+				Eventually(func() []logrus.Entry {
```

Comment:
Is this change required?

Reply:
Yes, it breaks the tests because one of the updates was the logrus